graylog2-web-interface/src/views/components/visualizations/heatmap/HeatmapVisualization.jsx

```diff
@@ -10,10 +11,66 @@ import type { ChartDefinition, ExtractedSeries } from '../ChartData';
 import GenericPlot from '../GenericPlot';
 import { chartData } from '../ChartData';
 
-const _generateSeries = (type, name, x, y, z): ChartDefinition => ({ type, name, x: y, y: x, z, transpose: true });
+const BG_COLOR = '#440154';
+
+const COLORSCALE = [
+  [0.00, '#440154'],
+  [0.05, '#481567'],
+  [0.10, '#483677'],
+  [0.15, '#453781'],
+  [0.20, '#404788'],
+  [0.30, '#39568c'],
+  [0.35, '#33638d'],
+  [0.40, '#2d708e'],
+  [0.45, '#287d8e'],
+  [0.50, '#238a8d'],
+  [0.55, '#1f968b'],
+  [0.60, '#20a387'],
+  [0.65, '#29af7f'],
+  [0.70, '#3cbb75'],
+  [0.75, '#55c667'],
+  [0.80, '#73d055'],
+  [0.85, '#95d840'],
+  [0.90, '#b8de29'],
+  [0.95, '#dce319'],
+  [1.00, '#fde725'],
+];
+
+const _generateSeries = (type, name, x, y, z, idx, total, config): ChartDefinition => {
+  const xAxisTitle = get(config, ['_value', 'rowPivots', idx, 'field']);
+  const yAxisTitle = get(config, ['_value', 'columnPivots', idx, 'field']);
+  const seriesTitle = get(config, ['_value', 'series', idx, '_value', 'function']);
```

Comment:
One thing, that bothers me, what should we display, when multiple pivots / series are defined. Something like: `dayOfTheWeek / weekday: 2` would be too huge for the info box. Regarding the series, we can't display the correct metric, when multiple are defined. Should we may just show `value:` instead?